src/components/Toggle/Toggle.tsx

```diff
@@ -45,7 +45,7 @@ function Toggle({
 
   function onClick(e: React.MouseEvent<HTMLButtonElement>) {
     // '`onChange` callback for this component
-    if (onChange) onChange(active)
+    if (onChange) onChange(!active)
```

Comment:
onChange was given false when Toggle was pressed and true when it was not, it should be `!active`

Reply:
ah yea, this looks better.